frontend/src/components/GrantAccess.vue

```diff
@@ -1,34 +1,40 @@
 <script setup lang="ts">
-import PageTitle from '@/components/common/PageTitle.vue';
 import { ApiServiceFactory } from '@/services/ApiServiceFactory';
-import { selectedApplication } from '@/store/ApplicationState';
+import {
+    selectedApplication,
+    selectedApplicationDisplayText,
+    grantAccessFormData,
+} from '@/store/ApplicationState';
 import type {
     FamApplicationRole,
     FamForestClient,
     FamUserRoleAssignmentCreate,
 } from 'fam-api';
 import { onMounted, ref } from 'vue';
-import { useToast } from 'vue-toastification';
 import { Form as VeeForm, Field, ErrorMessage } from 'vee-validate';
 import router from '@/router';
 import { number, object, string } from 'yup';
+import Button from 'primevue/button';
```

Comment:
Since we already created a reusable button component under the common folder, let's change to use that one. Although we need to add more properties in that reusable button component.

Reply:
think I'll create another ticket for this, don't want to add more work to this ticket, and it's not urgent